backend/appwrite-functions/announcement/ancm-function/main.py

```diff
@@ -0,0 +1,212 @@
+# SPDX-License-Identifier: MIT
+# SDPX-FileCopyrightText: 2021 Que Le <b.le@tu-berlin.de>
+
+import os
+import json
+import time
+import traceback
+import sys
+
+from appwrite.client import Client
+from appwrite.services.database import Database
+
+
+def init_client():
+    # Initialize the Appwrite client
+    client = Client()
+    client.set_endpoint(os.getenv("APPWRITE_ENDPOINT"))
+    client.set_project(os.getenv("APPWRITE_FUNCTION_PROJECT_ID"))
+    client.set_key(os.getenv("APPWRITE_API_KEY"))
+
+    return client
+
+
+def print_failure_message(message: str):
+    print(json.dumps({"status": "failure", "message": message}))
+
+
+def get_collection_id(database: Database):
+    collection_id = -1
+    listCollection = database.list_collections()
+    for collection in listCollection["collections"]:
+        if collection["name"] == "Announcements":
+            collection_id = collection["$id"]
+            break
+    if collection_id == -1:
+        print_failure_message("Internal error")
+        # TODO: hide this information from client. Now we need it for debugging. May be log system?
+        print("'Announcements' collection not found!", file=sys.stderr)
+        exit()
+    return collection_id
+
+
+def handleAdd(client_payload: dict, database: Database, collection_id: str):
```

Comment:
The name is not very clear.
How about:
`addAnnouncement`
`updateAnnouncement`
`removeAnnouncement`
`retrieveAnnouncement`

Reply:
renamed